src/components/Helpers/Tooltip/Tooltip.stories.mdx

```diff
@@ -0,0 +1,31 @@
+import { Meta, Story, Props, Preview } from "@storybook/addon-docs/blocks";
+import { Tooltip } from "./Tooltip";
+
+<Meta title="Tooltip/Info" component={Tooltip} />
+
+# Tooltip
```

Comment:
Should group the entire tooltip folder with the buttons, it's technically a button with tooltip. `IconWithTooltip` can be it's name or something